Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/BlendGraphWidget.cpp

```diff
@@ -1292,6 +1292,7 @@ namespace EMStudio
             }
 
             EMotionFX::AnimGraphNodeGroup* newGroup = animGraph->GetNodeGroup(animGraph->GetNumNodeGroups() - 1);
+            newGroup->SetParentNodeId(GetActiveGraph()->GetAnimGraphModel().GetFocus().data(AnimGraphModel::ROLE_ID).value<EMotionFX::AnimGraphNodeId>());
```

Comment:
suggestion: It might be worth adding a brief comment here to explain what is happening here and why

I'd also suggest breaking the line slightly as it's a long one 😅 

```
newGroup->SetParentNodeId(
    GetActiveGraph()->GetAnimGraphModel().GetFocus().data(AnimGraphModel::ROLE_ID).value<EMotionFX::AnimGraphNodeId>());
```

Reply:
Done